dstyle.dd

```diff
@@ -25,6 +25,29 @@ $(UL
     $(LI Use spaces instead of hardware tabs.)
 
     $(LI Each indentation level will be four columns.)
+
+    $(LI Braces should be on their own line. There are a few exceptions to this
```

Comment:
I removed the "additional requirements for Phobos" heading. There are no additional requirements for Phobos. The whole thing is a D style guide, and Phobos is expected to abide by it. Other projects can create their own style guide by picking and choosing what they wish - this one is the official one for Phobos.